Source/Core/Common/MsgHandler.cpp

```diff
@@ -106,20 +106,9 @@ std::string GetStringT(const char* string)
   return s_str_translator(string);
 }
 
-// This is the first stop for gui alerts where the log is updated and the
-// correct window is shown
-bool MsgAlert(bool yes_no, MsgType style, const char* format, ...)
+static bool ShowMessageAlert(const char* caption, const char* text, bool yes_no, MsgType style)
```

Comment:
Could we take a `std::string_view` instead of a `const char*` for `text`? (so fmt doesn't have to recalculate the string length unnecessarily)